### PR TITLE
update all plugins + pin go to go 1.17

### PR DESCRIPTION
**Pull request by azr**

### CODE REVIEW

**Review by SwampDragons — COMMENTED** [review]
😱 merge when you're ready.

**Review by nywilken — APPROVED** [review]
Nice - looks good. Once merged can you bump the go version for the packer release pipeline as well https://github.com/hashicorp/packer-releases/blob/02233defe64ab84946431045cb1862e3313d7742/.circleci/config.yml#L17

File: go.sum [hunk 1]
```diff
@@ -733,31 +794,26 @@ github.com/hashicorp/packer v1.6.7-0.20210217093213-201869d627bf/go.mod h1:+EWPP
 github.com/hashicorp/packer v1.7.0/go.mod h1:3KRJcwOctl2JaAGpQMI1bWQRArfWNWqcYjO6AOsVVGQ=
 github.com/hashicorp/packer v1.7.1/go.mod h1:ApnmMINvuhhnfPyTVqZu6jznDWPVYDJUw7e188DFCmo=
 github.com/hashicorp/packer v1.7.2/go.mod h1:c/QB/DWK5fSdtNWrTb9etWacmbm01UY23ZILpGundCY=
-github.com/hashicorp/packer v1.7.4/go.mod h1:uD7xkJ8yYOLUHEIOasoOXrIEmpdOibOIdwHUYroi6zc=
```
Comment (azr) [inline go.sum:736]: 🤔

File: command/vendored_plugins.go [hunk 2]
```diff
@@ -6,7 +6,7 @@ import (
 	// Previously core-bundled components, split into their own plugins but
 	// still vendored with Packer for now. Importing as library instead of
 	// forcing use of packer init, until packer v1.8.0
-	exoscaleimportpostprocessor "github.com/exoscale/packer-plugin-exoscale/post-processor/exoscale-import"
```
Comment (azr) [inline command/vendored_plugins.go:9]: turn out we had moved out the builder but not the importer !

File: go.sum [hunk 3]
```diff
@@ -723,52 +708,34 @@ github.com/hashicorp/mdns v1.0.1/go.mod h1:4gW7WsVCke5TE7EPeYliwHlRUyBtfCwuFwuMg
...
-github.com/hashicorp/packer v1.6.7-0.20210208125835-f616955ebcb6/go.mod h1:7f5ZpTTRG53rQ58BcTADuTnpiBcB3wapuxl4sF2sGMM=
-github.com/hashicorp/packer v1.6.7-0.20210217093213-201869d627bf/go.mod h1:+EWPPcqee4h8S/y913Dnta1eJkgiqsGXBQgB75A2qV0=
-github.com/hashicorp/packer v1.7.0/go.mod h1:3KRJcwOctl2JaAGpQMI1bWQRArfWNWqcYjO6AOsVVGQ=
-github.com/hashicorp/packer v1.7.1/go.mod h1:ApnmMINvuhhnfPyTVqZu6jznDWPVYDJUw7e188DFCmo=
-github.com/hashicorp/packer v1.7.2/go.mod h1:c/QB/DWK5fSdtNWrTb9etWacmbm01UY23ZILpGundCY=
-github.com/hashicorp/packer v1.7.4/go.mod h1:uD7xkJ8yYOLUHEIOasoOXrIEmpdOibOIdwHUYroi6zc=
 github.com/hashicorp/packer-plugin-alicloud v1.0.0 h1:f1wyIqgSQQlpYs116Auq/FLYtrW3GbpTzxXxZWmNc1A=
 github.com/hashicorp/packer-plugin-alicloud v1.0.0/go.mod h1:wDWvfgD8JWdrGnAzO6SkcJJiAL+64SCEVY+DWcefNas=
-github.com/hashicorp/packer-plugin-amazon v0.0.1/go.mod h1:12c9msibyHdId+Mk/pCbdRb1KaLIhaNyxeJ6n8bZt30=
-github.com/hashicorp/packer-plugin-amazon v1.0.0/go.mod h1:A4cjwkca0op3y/IWV3z9vTCHeJY2nLwBnwRxRFlFw6Y=
```
Comment (azr) [inline go.sum:741]: Yes, I managed to get rid of those !

**Pull request merged**